tests/dbus/test-client

```diff
@@ -57,6 +66,27 @@ main()
     sudo "${CMAKE_BINARY_DIR}"/src/agent/otbr-agent -d7 -I wpan0 "spinel+hdlc+forkpty://$(command -v ot-rcp)?forkpty-arg=1" &
     trap on_exit EXIT
     sleep 5
+
+    sudo "${CMAKE_BINARY_DIR}"/third_party/openthread/repo/src/posix/ot-ctl ifconfig up
+    sudo "${CMAKE_BINARY_DIR}"/third_party/openthread/repo/src/posix/ot-ctl thread start
+
+    sleep 10
+
+    sudo gdbus call --system --dest io.openthread.BorderRouter.wpan0 --method=io.openthread.BorderRouter.UpdateMeshCopTxt --object-path /io/openthread/BorderRouter/wpan0 "[('vn',[118,101,110,100,111,114])]"
+    sleep 5
+    service="$(scan_meshcop_service)"
+    grep --binary-files=text "vn=vendor" <<<"${service}"
+
+    sudo gdbus call --system --dest io.openthread.BorderRouter.wpan0 --method=io.openthread.BorderRouter.UpdateMeshCopTxt --object-path /io/openthread/BorderRouter/wpan0 "[]"
+    sleep 5
+    service="$(scan_meshcop_service)"
+    grep --binary-files=text "vn=OpenThread" <<<"${service}"
+
+    sudo gdbus call --system --dest io.openthread.BorderRouter.wpan0 --method=io.openthread.BorderRouter.UpdateMeshCopTxt --object-path /io/openthread/BorderRouter/wpan0 "[('A',[97,98,99])]"
+    sleep 5
+    service="$(scan_meshcop_service)"
+    grep --binary-files=text "A=abc" <<<"${service}"
+
```

Comment:
Put the test cases in a separate source file?

Reply:
I've extracted this part as a function.